src/Grids/vertically_stretched_rectilinear_grid.jl

```diff
@@ -323,6 +323,29 @@ all_z_nodes(::Type{Face}, grid::VerticallyStretchedRectilinearGrid) = grid.zᵃ
 # Get minima of grid
 #
 
-min_Δx(grid::VerticallyStretchedRectilinearGrid) = grid.Δx
-min_Δy(grid::VerticallyStretchedRectilinearGrid) = grid.Δy
-min_Δz(grid::VerticallyStretchedRectilinearGrid) = minimum(view(grid.Δzᵃᵃᶜ, 1:grid.Nz))
+function min_Δx(grid::VerticallyStretchedRectilinearGrid)
+    topo = topology(grid)
+    if topo[1] == Flat
+        return Inf
+    else
+        return grid.Δx
+    end
+end
+
+function min_Δy(grid::VerticallyStretchedRectilinearGrid)
```

Comment:
Also, the three min functions you defined looks like they will work but I wonder if they could be combined into one?  Not necessary here but more a question on style

Reply:
Same answer here! It's a bit verbose but I think readability makes up for it.

Reply:
In principle, having two functions defined almost identically in two different scripts seems less than ideal.  If ever you have to change the function, then you have to change it in two places.  This case is pretty straightforward so I don't imagine the functions will change much though.